dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/ShutdownServiceImpl.java

```diff
@@ -140,7 +140,7 @@ public Object call() {
                 final CountDownLatch latch = new CountDownLatch(1);
                 try {
 
-                    String jobId = "";
+                    final String jobId = "";
```

Comment:
should this be job.getJobId()?  and a few lines lower?  A final empty string does not seem to be a job ID